ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/SvarUtWebServiceClientImpl.java

```diff
@@ -0,0 +1,19 @@
+package no.difi.meldingsutveksling.ks;
+
+import org.springframework.ws.client.core.support.WebServiceGatewaySupport;
+
+public class SvarUtWebServiceClientImpl  extends WebServiceGatewaySupport implements SvarUtWebServiceClient {
+    public String sendMessage(Forsendelse forsendelse) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)